code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -355,6 +355,12 @@ GLOBAL_LIST_EMPTY(roundstart_station_closets)
 			. += span_notice("Swipe your PDA with an ID card/Just ID to change access levels.")
 			. += span_notice("Use multitool to [access_locked ? "unlock" : "lock"] the access panel.")
 
+//BUBBERSTATION EDIT START - LOCKER CONTENT EXAMINING
+	if(isobserver(user))
+		. += span_info("It contains: [english_list(contents)].")
+		investigate_log("had its contents examined by [user] as a ghost.", INVESTIGATE_GHOST)
+//BUBBERSTATION EDIT END - LOCKER CONTENT EXAMINING
```

Comment:
Can you override the examine proc on lockers in a zubbers folder, for example in modular_zubbers/code/objects/structures/crate_lockers/closets.dm , and overide the examine() proc there, and if you call parent at the start of that, it makes it nice and modular